### ISSUE TITLE
Running HiBench workloads in Yarn-cluster mode

### ISSUE THREAD

**Issue by srikrishnacancun** [intel-hadoop/HiBench#197]
Is HiBench supported for Yarn-Cluster mode for any of the workloads .?

**Comment by srikrishnacancun** [comment]
Look for terrasort ,KMeans, PageRank workloads support for Yarn-Cluster mode

**Comment by yncxcw** [comment]
yes, the default HiBench running on Spark is under Yarn-Cluster mode

**Comment by srikrishnacancun** [comment]
Hi

When I change the Hibench bench conf file to point to Yarn-Cluster and ran
the wordcount workload and observed the Spark UI, I don't see the driver
program invoked on one of the datanodes.  Is there a different procedure.?

Looking forward in hearing back.

Thanks

Srikrishna

**Comment by carsonwang** [comment]
Hi, currently Yarn-cluster mode is not supported by HiBench.
